crates/core/src/auth/identity.rs

```diff
@@ -8,6 +8,8 @@ use std::collections::HashSet;
 use std::time::Duration;
 use std::time::SystemTime;
 
+use super::token_validation::TokenValidationError;
+
 #[serde_with::serde_as]
 #[derive(Debug, Serialize, Deserialize)]
 pub struct SpacetimeIdentityClaims {
```

Comment:
Is this still used? Is this being removed in another PR?

Reply:
I'm not sure, but I can add that to the list of clean up needed to follow up on this PR.